hotness_schema/hotness_schema/messages.py

```diff
@@ -158,14 +161,24 @@ def summary(self):
         Returns:
             (str): Short description of the message.
         """
-        return "the-new-hotness filed a bug on '{}'".format(self.package)
+        return "the-new-hotness filed a bug on '{}'".format(self.packages)
 
     @property
-    def package(self):
+    def packages(self):
         """
-        Return package name.
+        Return package names.
 
         Returns:
-            (str): Package name.
+            (str): Package names delimited by ','.
```

Comment:
The signature of the `packages()` method on the `Message` class is to return a list.

Reply:
I didn't noticed, that `Message` class has already `packages()` method. I will fix this.